www/src/pages/ru/deployment/docker.md

```diff
@@ -129,7 +129,7 @@ CMD ["node", "server.js"]
 > **_Заметки_**
 >
 > - _Эмуляция `--platform=linux/amd64` может не быть необходимой после перехода на Node 18._
-> - _Посмотрите [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) чтобы понять, почему `libc6-compat` может быть необходим._
+> - _Посмотрите [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) чтобы понять, почему `libc6-compat` может быть необходим._ -_Использование пакетов базирующихся на Alpine 3.17 [может привести к ошибке](https://github.com/t3-oss/create-t3-app/issues/975). Установка `engineType = "binary"` решает проблему с Alpine 3.17, [но имеет связанные с этим затраты производительности](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)._
```

Comment:
```suggestion
> - _Посмотрите [`node:alpine`](https://github.com/nodejs/docker-node/tree/b4117f9333da4138b03a546ec926ef50a31506c3#nodealpine) чтобы понять, почему `libc6-compat` может быть необходим._ 
> -_Использование образов, основанных на Alpine 3.17 [может привести к проблемам с Prisma](https://github.com/t3-oss/create-t3-app/issues/975). Установка `engineType = "binary"` решает проблему с Alpine 3.17, [но имеет связанные с этим затраты производительности](https://www.prisma.io/docs/concepts/components/prisma-engines/query-engine#the-query-engine-at-runtime)._
```

Reply:
![image](https://user-images.githubusercontent.com/23616925/212692747-cc1a74e5-3dec-44d8-b8a4-0e0e7c257ee2.png)
I think this result was not expected. It should have 4 bullet points.